src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -297,6 +297,9 @@
   <data name="net_http_unix_invalid_credential" xml:space="preserve">
     <value>libcurl does not support different credentials for different authentication schemes</value>
   </data>
+  <data name="net_http_unix_https_libcurl_no_versioninfo" xml:space="preserve">
+    <value>The call to GetCurlVersionInfo failed unexpectedly.</value>
```

Comment:
I wonder if this should be "curl_version_info" rather than "GetCurlVersionInfo", since curl_version_info is part of the libcurl public interface whereas GetCurlVersionInfo is an internal implementation detail of our solution.